platform/cosmos/model.go

```diff
@@ -0,0 +1,32 @@
+package cosmos
+
+type Tx struct {
+	BlockHeight uint64 `json:"height"`
+	Timestamp   string `json:"timestamp"`
+	Hash        string `json:"txhash"`
+	TxData      struct {
+		TxType  string `json:"type"`
+		TxValue struct {
+			TxMessage struct {
+				TxContents []TxContents `json:"value"`
```

Comment:
does not look good this way, can you split into separate models in the file?

Reply:
I'll finish up the PR, how urgent is it?